dialogue-core/src/main/java/com/palantir/dialogue/core/Channels.java

```diff
@@ -46,16 +46,17 @@ public static Channel create(
                 .map(channel -> new DeprecationWarningChannel(channel, clientMetrics))
                 // TracedChannel must wrap TracedRequestChannel to ensure requests have tracing headers.
                 .map(TracedRequestChannel::new)
-                .map(channel -> new TracedChannel(channel, "Concurrency-Limited Dialogue Request"))
                 .map(ContentDecodingChannel::new)
                 .map(concurrencyLimiter(config))
                 .collect(ImmutableList.toImmutableList());
 
         LimitedChannel limited = nodeSelectionStrategy(config, limitedChannels);
         Channel channel = new QueuedChannel(limited, DispatcherMetrics.of(config.taggedMetricRegistry()));
+        channel = new TracedChannel(channel, "Dialogue-request-attempt");
```

Comment:
It’s a little odd that we provide spans for attempts which may or may not actually make it to an http client (blacklisting, concurrency limiting, etc).
Perhaps we should wrap the client channel as well to show real http requests?

Reply:
I'm not sure that wrapping the client channel would show up in our tests, I think we need to add a bit of tracing-aware code to queued channel.

Reply:
Ya totally agree, I decided to go down this path since it seemed nice from a users perspective to always be able to see N request attempts regardless oh what happened during the attempt. I can bring back https://github.com/palantir/dialogue/pull/385/files/a82380f80f878ed1e33394317d7b640f79f3cf11#diff-3e072c17c066c68f3570b3b5f19ed29dL49 to show when "real" requests are made

Reply:
👍